aml-pipeline/.amlignore

```diff
@@ -0,0 +1,12 @@
+## This file was auto generated by the Azure Machine Learning Studio. Please do not remove. 
```

Comment:
Please update the pkg version in setup.py

Reply:
Should we update to version '0.0.6'?